pkg/didcomm/messaging/service/basic/basicmsgsvc.go

```diff
@@ -4,7 +4,7 @@
  *
  * SPDX-License-Identifier: Apache-2.0
  * /
- *
+ *.
```

Comment:
This is change is happening several times. I won't add further comments but I see them below too.